### PR TITLE
Add setup guide for debian docker stretch Nvidia harware decode

### PR DESCRIPTION
**Pull request by Tontonjo**
I've spent some time to get hardware transcoding working on docker container:
here are the steps i had to went trough to get it working.

Hope it's ok :-)

### CODE REVIEW

File: general/administration/hardware-acceleration.md [hunk 1]
```diff
@@ -87,6 +87,127 @@ services:
...
+-e NVIDIA_VISIBLE_DEVICES=all \
+--runtime=nvidia \
+--gpus all \
+```
+A complete run command would looks like:
+```
+docker run -d \
+--name=jellyfin \
+-e PUID=1000 \
+-e PGID=100 \
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md]: Can you run id in your container? AFAIK, the jellyfin image doesn't support user environmental variables.

Comment (Tontonjo) [reply general/administration/hardware-acceleration.md]: My bad - this can be removed.
Was usefull when using linuxserver.io container.

Comment (Artiume) [reply general/administration/hardware-acceleration.md]: so if you want to run the user in the container use --user 1000:1000

you might have to add the user to the video group? depends on your os and who owns the gfx cards. for my rpi4, i had to do 

`usermod -aG video user`

Comment (Tontonjo) [reply general/administration/hardware-acceleration.md]: This was not needed on my side to add the user to any group
As the env. settings for the user was not correctly set i guess it was running as root so i had no problem.
Can add infos about this.

Comment (Artiume) [reply general/administration/hardware-acceleration.md]: sure. any additional info you can add will always be welcome. I dont have an nvidia card so I rely on others for the documentation :)

File: general/administration/hardware-acceleration.md [hunk 2]
```diff
@@ -87,6 +87,136 @@ services:
       # RPi 4
       - /dev/vchiq:/dev/qchiq
 ```
+## Debian Docker Nvidia Hardware acceleration:
+
+In order to achieve Hardware decode inside docker containers, some steps are needed:
+Thoses steps where wrote for Debian Stretch.
+
+Prerequisits:
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md]:
```suggestion
Prerequisites:
```

File: general/administration/hardware-acceleration.md [hunk 3]
```diff
@@ -87,6 +87,136 @@ services:
       # RPi 4
       - /dev/vchiq:/dev/qchiq
 ```
+## Debian Docker Nvidia Hardware acceleration:
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md]:
```suggestion
## Debian Docker Nvidia:
```

File: general/administration/hardware-acceleration.md [hunk 4]
```diff
@@ -87,6 +87,136 @@ services:
...
+
+In order to achieve Hardware decode inside docker containers, some steps are needed:
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md]:
```suggestion
In order to achieve Hardware acceleration using docker, some steps are needed:
```

File: general/administration/hardware-acceleration.md [hunk 5]
```diff
@@ -87,6 +87,136 @@ services:
...
+Thoses steps where wrote for Debian Stretch.
+
+Prerequisits:
+https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0)
+
+    GNU/Linux x86_64 with kernel version > 3.10
+    Docker >= 1.12
+    NVIDIA GPU with Architecture > Fermi (2.1)
+    NVIDIA drivers ~= 361.93 (untested on older versions)
```
Comment (Artiume) [inline general/administration/hardware-acceleration.md]: Just curious, have you tried these drivers? 

https://github.com/keylase/nvidia-patch

Comment (Tontonjo) [reply general/administration/hardware-acceleration.md]: Yes i did patched my drivers: looks to be working as, if understood the limitation of consumer GPUS, i could get 3 transcodes in parallel

File: general/administration/hardware-acceleration.md (hunk 4 above)
Comment (Tontonjo) [reply general/administration/hardware-acceleration.md]: corrected

**Comment by anthonylavado** [comment]
Ahh you did it! Thank you so much!!

**Pull request merged**